pkg/resources/create_test.go

```diff
@@ -139,22 +139,12 @@ func TestCreateResource(t *testing.T) {
 	logger, _ := logging.NewLogger("", "")
 
 	tests := []struct {
-		name     string
-		resource pipelinev1.PipelineResource
-		want     pipelinev1.PipelineResource
+		name string
+		json []byte
+		want pipelinev1.PipelineResource
 	}{{
 		name: "PipelineResource without namespace",
-		resource: pipelinev1.PipelineResource{
-			TypeMeta: metav1.TypeMeta{
-				APIVersion: "tekton.dev/v1alpha1",
-				Kind:       "PipelineResource",
-			},
-			ObjectMeta: metav1.ObjectMeta{
-				Name:   "my-pipelineresource",
-				Labels: map[string]string{"woriginal-label-1": "label-1"},
-			},
-			Spec: pipelinev1.PipelineResourceSpec{},
-		},
+		json: json.RawMessage("{\"kind\":\"PipelineResource\",\"apiVersion\":\"tekton.dev/v1alpha1\",\"metadata\":{\"name\":\"my-pipelineresource\",\"creationTimestamp\":null,\"labels\":{\"woriginal-label-1\":\"label-1\"}},\"spec\":{\"type\":\"\",\"params\":[{\"name\":\"foo\",\"value\":\"bar\\r\\nbaz\"}]},\"status\":{}}"),
```

Comment:
Could we use raw strings `` here...then we could lose the escaping around the quotes `"`?

Reply:
good call! :D